solver/src/solver/http_solver.rs

```diff
@@ -18,6 +19,7 @@ use shared::{
     price_estimate::{PriceEstimating, PriceEstimationError},
     token_info::{TokenInfo, TokenInfoFetching},
 };
+use std::ops::Div;
```

Comment:
This shouldn't be needed.

Reply:
My IDE said it was, but I will remove it.

Reply:
It was needed because you used `a.div(b)` instead of `a / b`, but see suggestion :point_down: